### PR TITLE
Improve selectizer options

### PR DESCRIPTION
**Pull request by jadb**
by allowing multi-select lists and control over some more options:

- `maxOptions`: using `data-max-options`
- `maxItems`: using `data-max-items`
- `hideSelected`: using `data-hide-selected`
- `closeAfterSelect`: using `data-close-after-select`

### CODE REVIEW

File: webroot/js/local.js [hunk 1]
```diff
@@ -41,8 +41,11 @@ $(document).on('ready', function() {
     $('input.autocomplete, select.autocomplete').each(function (i, e) {
         e = $(e);
         e.selectize({
-            maxItems: 1,
-            create: !$(e).data('exact-match'),
+            maxItems: e.data('max-items') || 1,
```
Comment (ADmad) [inline webroot/js/local.js]: Too bad selectize doesn't automatically read these data attributes.

**Comment by lorenzo** [comment]
How can you use these options?

**Comment by jadb** [comment]
The same way the other (i.e. url, exact-match, etc.) were used:

```php
'fieldKey' => ['data-max-items' => 10]
```

**Comment by lorenzo** [comment]
Would you mind adding that to the docs?

File: webroot/js/local.js (hunk 1 above)
Comment (jippi) [inline webroot/js/local.js]: i would really prefer to go back to https://select2.github.io/ :)

**Pull request merged**